vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -363,6 +363,8 @@ components:
         memory_actual_size:
           type: integer
           format: int64
+        device_tree:
+          type: object
```

Comment:
I think we should also specify the kind of object a `device_tree` may contain, i.e.:

```yaml
      type: object
      additionalProperties:
        $ref: '#/components/schemas/DeviceNode'
```

Reply:
Let me look into this. I'm not sure about whether the `DeviceNode` is "stable" or can even be safely represented like this.

Reply:
Done, but we can't specify `DeviceNode::resources` beyond being a generic associative array.